examples/webgl_curvature_estimation.html

```diff
@@ -0,0 +1,530 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - shader - curvature [ninja]</title>
+		<div class="label" style="position: absolute;left: 20%;bottom: 10%;">Raw curvature values</div>
```

Comment:
Hmm, what are these `<div>`s for?

Reply:
they are the descriptions under the two scenes   ("Raw curvature values" and "Curvature driving specular + texture")

Reply:
Oh, I'll move them to <body>